### PR TITLE
Release with taskboot.

### PR DESCRIPTION
**Pull request by La0**
Needs https://github.com/mozilla/community-tc-config/pull/173

### CODE REVIEW

File: .taskcluster.yml [hunk 1]
```diff
@@ -57,3 +57,70 @@ tasks:
...
+              PROJECT_NAME: rust-code-analysis
+            command:
+              - "/bin/bash"
+              - "-cx"
+              - "git clone --recursive --quiet ${repository} &&
+                 cd rust-code-analysis &&
+                 git -c advice.detachedHead=false checkout ${head_rev} &&
+                 cargo build --release &&
+                 cd target/release &&
+                 tar -zvcf /build.tar.gz rust-code-analysis"
```
Comment (marco-c) [inline .taskcluster.yml]: This is going to be different than the current directory structure. I guess the only two users are bugbug and microannotate for now, so we can live with the change.

Comment (La0) [reply .taskcluster.yml]: I can replicate the current one

**Comment by La0** [comment]
https://github.com/mozilla/rust-code-analysis/releases/tag/test-0.0.5

**Comment by La0** [comment]
https://github.com/mozilla/rust-code-analysis/releases/tag/test-0.0.6 with the folder in the payload

**Pull request merged**